configuration.yaml

```diff
@@ -72,3 +72,15 @@ TargetSafety:
   toxcast: gs://otar001-core/TargetSafety/data_files/toxcast/ToxCast_2021-08-17.tsv
   aopwiki: gs://otar001-core/TargetSafety/data_files/aopwiki/aopwiki-2023-01-20.json
   outputBucket: gs://otar001-core/TargetSafety/json
+OT_CRISPR:
+  data_directory: gs://otar013-ppp/ot_crispr/input_data
+  config: gs://otar013-ppp/PPP-evidencie-configuration/ot_crispr_config.json
+  outputBucket: gs://otar013-ppp/ot_crispr
+ValidationLab:
+  data_directory: /Users/dsuveges/project_data/validation_lab/2022.05/PPP_OTVL_May2022
```

Comment:
I don't think this local path is supposed to be present here?

(Also the same for the Encore section)

Reply:
Oh, I need to upload the input files to the google bucket.